api/environments/urls.py

```diff
@@ -142,4 +143,9 @@
         create_segment_override,
         name="create-segment-override",
     ),
+    path(
+        "<int:environment_pk>/edge-identity-overrides",
```

Comment:
I'm not sure whether to use the `pk` here or the API key. I definitely prefer using the pk, and would like to move to using the `pk` everywhere in the future so I figured this would at least not add further tech debt. It does, however, cause some consistency issues.